### PR TITLE
Add configuration for `comment-ops`

### PR DESCRIPTION
**Pull request by timja**
With https://github.com/timja/github-comment-ops/issues/69 commands are being switched to off by default so let's explicitly enable what we want here

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6985"></a>

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

File: .github/comment-ops.yml
Comment (daniel-beck) [inline .github/comment-ops.yml:5]: The use case for these is unclear to me given the large number of maintainers.

**Review by daniel-beck — APPROVED** [review]
+0.5; reviewer makes sense, the rest less so.

File: .github/comment-ops.yml
Comment (timja) [reply .github/comment-ops.yml:5]: you can have saved replies that do a few things at once,

e.g. 

```
label web-ui, rfe

reviewer jenkinsci/sig-ux
```
and also this allows non maintainers to setup the labels to save maintainers doing it.

Comment (daniel-beck) [reply .github/comment-ops.yml:5]: > that do a few things at once

Bot commands only makes sense for people who cannot use the built-in features anyway (unless you like spamming watchers).

> this allows non maintainers to setup the labels to save maintainers

Which assumes that people are familiar with how we label things. I already explained in the UX meeting that I've experienced that differently in the subprojects I maintain. Given the large number of active maintainer+ level contributors, the case for this is weak in this repo IMO.

Comment (timja) [reply .github/comment-ops.yml:5]: >  Given the large number of active maintainer

We don't have what I would call large =/. As one of the (~3) people doing this it would be less work for me if it has been done.

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
